qemu/tests/cfg/blockdev_inc_backup_resize.cfg

```diff
@@ -0,0 +1,41 @@
+# Storage backends:
+#   filesystem, gluster_direct, ceph
+# The following testing scenario is covered:
+#   resize a qcow2 image with persistent bitmap stored on it
+
+
+- blockdev_inc_backup_resize:
+    only Linux
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_inc_backup_resize
+    virt_test_type = qemu
+    images += " data"
+    source_images = "data"
+    image_backup_chain_data = "base"
+    remove_image_data = yes
+    force_create_image_data = yes
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data = 2G
+    image_size_base = 2G
+
+    image_format_data = qcow2
+    image_format_base = qcow2
+
+    image_name_data = data
+    image_name_base = base
+    rebase_mode = unsafe
+    check_bitmaps = 'bitmaps:'
+    storage_type_default = "directory"
+
+    # settings for block_resize
+    no RHEL.4
+    only filesystem ceph gluster_direct
+    extend_ratio = 1.5
+    shrink_ratio = 0.75
```

Comment:
Better to be 1, otherwise, there is risk since the original data disk might be out of space.

Reply:
According to my test results, 0.75 works fine, for only 20M data is written to a 2G*0.75 disk. 
If you hit the 'out of space' issue on ppc, I'm afraid we have to enlarge the data disk to 3G or 4G, because shrinking the disk is an assertion (2G --> 2G*0.75), thank you.